setup.py

```diff
@@ -70,7 +70,7 @@
     },
     # Packaging
     package_dir={"": "src"},
-    packages=find_packages("src") + namespace_packages,
+    packages=find_namespace_packages(where="src"),
```

Comment:
This needs to be documented why we use this here, as it is not obvious (sander had to explain)